src/System.Management.Automation/engine/remoting/server/ServerRunspacePoolDriver.cs

```diff
@@ -133,7 +138,8 @@ internal class ServerRunspacePoolDriver : IRSPDriverInvoke
             bool isAdministrator,
             RemoteSessionCapability serverCapability,
             Version psClientVersion,
-            string configurationName)
+            string configurationName,
+            string initialLocation)
 #else
```

Comment:
It looks like you have an old source snapshot there are no conditional compilation directives here in the current code base.  You may want to merge the latest source.

Reply:
I rebased from master and I fixed the merge conflicts in 95ab95d2d5c043a1fbc3ba78f1ff94c9e45d94af